functions-and-scope.js

```diff
@@ -27,9 +37,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // cumLaude([6, 4, 5]) geeft 0
 // cumLaude([8, 9, 4, 6, 10]) geeft 3
 
-
-
-
+function cumLaude(ArrayList) {
```

Comment:
Variabelen camelCase schrijven
```suggestion
function cumLaude(arrayList) {
```